website/docs/components/text/partials/code/component-api.md

```diff
@@ -0,0 +1,87 @@
+  </C.Property>
+  <C.Property @name="yield">
+    Elements passed as children are yielded to the content of the `<@tag>` HTML element.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### Text::Body
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="size" @type="string" @values={{array "300" "200" "100" }} @default="200">
+    The size of the `Body` text style.
+  </C.Property>
+  <C.Property @name="tag" @type="string" @default="span">
+    HTML tag to be used to render the text element.
+  </C.Property>
+  <C.Property @name="weight" @type="string" @values={{array "regular" "medium" "semibold" }} @default="regular">
+    The font weight of the text. If no `@weight` argument is provided, the component will inherit its weight from the parent container/context.
+  </C.Property>
+  <C.Property @name="align" @type="string" @values={{array "left" "center" "right" }}>
+    The alignemnt of the text. If no `@align` argument is provided, the component will inherit its alignment from the parent container/context.
+  </C.Property>
+  <C.Property @name="color" @type="string | CSS color" @values={{array "primary" "strong" "faint" "disabled" "high-contrast" "action" "action-hover" "action-active" "highlight" "highlight-on-surface" "highlight-high-contrast" "success" "success-on-surface" "success-high-contrast" "warning" "warning-on-surface" "warning-high-contrast" "critical" "critical-on-surface" "critical-high-contrast" }}>
+    The color of the text expressed as one of the possible [foreground color](/foundations/colors?tab=palette#foreground-1) names. As a fallback solution to handle special cases, a valid CSS color string (hex, rgb, rgba, etc.) is accepted too (in this case the color will be applied via inline style). If no `@color` argument is provided, the component will inherit its color from the parent container/context.
+  </C.Property>
+  <C.Property @name="yield">
+    Elements passed as children are yielded to the content of the `<@tag>` HTML element.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### Text::Code
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="size" @type="string" @values={{array "300" "200" "100" }} @default="200">
+    The size of the `Code` text style.
+  </C.Property>
+  <C.Property @name="tag" @type="string" @default="span">
+    HTML tag to be used to render the text element.
+  </C.Property>
+  <C.Property @name="weight" @type="string" @values={{array "regular" }} @default="regular">
```

Comment:
@heatherlarsen the `Code` variant at the moment supports only the `regular` font weight (to follow the styles in Figma). Do you think we should support also the `bold` weight (supported by the font-family we use) too, to be less restrictive? (there may be valid use cases in which a code snippet (or maybe part of it) may need to be visually made heavier for some reasons.

Reply:
Yes, let's add more flexibility there. I'll create a ticket to add that style in Figma as well.

Reply:
Updated.